package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "etebase",
-  "version": "0.43.0",
+  "version": "0.44.0",
```

Comment:
Don't bump the version, that will be done when we make a release.

Reply:
Sorry, that was just so I could publish for us (we need the fix 😁 )